targets/ath79-generic

```diff
@@ -89,6 +89,13 @@ device('enterasys-ws-ap3705', 'enterasys_ws-ap3705i', {
 
 -- GL.iNet
 
+device('gl.inet-gl-ar150', 'glinet_gl-ar150', {
+	factory = false,
+	manifest_aliases = {
+		'gl-ar150', -- upgrade from OpenWrt 19.07
```

Comment:
This can be dropped. The alias was only used for updates from LEDE 17.01, OpenWrt 18.06 already had the new name (and that is the oldest version that we still support upgrades from).

Reply:
Thanks, resolved in 7a47873e06467233caeb059dbffcd86143d8241e.